packages/perspective-jupyterlab/src/ts/renderer.ts

```diff
@@ -26,7 +36,7 @@ const baddialog = () => {
 
 export class PerspectiveDocumentWidget extends DocumentWidget<PerspectiveWidget> {
     constructor(options: DocumentWidget.IOptionsOptionalContent<PerspectiveWidget>, type: IPerspectiveDocumentType = "csv") {
-        super({content: new PerspectiveWidget("test"), context: options.context, reveal: options.reveal});
+        super({content: new PerspectiveWidget("Perspective"), context: options.context, reveal: options.reveal});
```

Comment:
This should be unique, no?

Reply:
I don't think so, why would it need to be?